test/integration/adoption_application_reviews_test.rb

```diff
@@ -33,7 +33,7 @@ class AdoptionApplicationReviewsTest < ActionDispatch::IntegrationTest
       get staff_adoption_application_reviews_path
 
       assert_response :success
-      AdopterApplication.all.each { |application| verify_application_elements application }
+      AdopterApplication.first(AdopterApplication::REVIEWS_PER_PAGE).each { |application| verify_application_elements application }
```

Comment:
Will need to also update this with a value after removing the constant from the model